### PR TITLE
baby_fuzzer_wasm fix

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: fuzzers/baby_fuzzer_wasm/src/lib.rs [hunk 1]
```diff
@@ -44,7 +44,7 @@ pub fn fuzz() {
     // No concurrency in WASM so these accesses are not racing.
     unsafe {
         RegistryBuilder::register::<MapFeedbackMetadata<u8>>();
-        RegistryBuilder::register::<ExecutionCountRestartHelperMetadata>();
+        RegistryBuilder::register::<StdRestartHelper >();
```
Comment (domenukk) [inline fuzzers/baby_fuzzer_wasm/src/lib.rs:47]: Wait, that's a different RestartHelper alltogether? That's the one that counts the target executions(?)

Comment (domenukk) [reply fuzzers/baby_fuzzer_wasm/src/lib.rs:47]: Can WASM even restart? :D

Comment (tokatoka) [reply fuzzers/baby_fuzzer_wasm/src/lib.rs]: > Wait, that's a different RestartHelper alltogether? That's the one that counts the target executions(?)

ExecutionCountRestartHelperMetadata was not even used in mutationalstage (the only stage that uses this metadata)

previously it was `Ok(true)` = always restart no matter what.
I then changed it to restart 3 times at max

Comment (domenukk) [reply fuzzers/baby_fuzzer_wasm/src/lib.rs]: We used ExecutionCountRestartHelper to resume the mutational stage (I still think this is more correct btw) but it performed worse in fuzzbench

File: fuzzers/baby_fuzzer_wasm/src/lib.rs [hunk 2]
```diff
@@ -44,7 +44,7 @@ pub fn fuzz() {
     // No concurrency in WASM so these accesses are not racing.
     unsafe {
         RegistryBuilder::register::<MapFeedbackMetadata<u8>>();
-        RegistryBuilder::register::<ExecutionCountRestartHelperMetadata>();
+        RegistryBuilder::register::<StdRestartHelper>();
```
Comment (domenukk) [inline fuzzers/baby_fuzzer_wasm/src/lib.rs:47]: Let's rename that to CountingRestartHelper as discussed

**Pull request merged**